### ISSUE TITLE
WAVE web accessibility evaluation

### ISSUE THREAD

**Issue by harpreetsgill** [harpreetsgill/portfolio#6]
## Description
The WAVE Web Accessibility Evaluation Tool checks if a website is accessible to people with disabilities. The portfolio has many errors and alerts that should be fixed.

## Steps to Reproduce
* Go to https://wave.webaim.org/
* Paste _https://gillharpreet.com/_ in the Web page address field
* Click the arrow to the right of the field

## Expected Result
There should be no errors, alerts and contrast errors under the Summary tab.

## Actual Result
The Summary tab shows 13 errors, 10 contrast errors and 6 alerts: 
![Top of the project page](https://i.ibb.co/vkqpz5j/wave.png)

## Configuration
This bug was found on the following devices and browsers:
![Configuration](https://i.ibb.co/CzPTtJD/config.png)
